INSTALL.md

```diff
@@ -225,3 +217,42 @@ Note that the OSM map tiles you see aren't created from your local database - th
 # Configuration
 
 After installing this software, you may need to carry out some [configuration steps](CONFIGURE.md), depending on your tasks.
+
+# Installing compiled shared library database functions
```

Comment:
Add `(optional)` to this title to highlight (and reassure) that it's not necessary for most people.